packages/p3x-onenote-app/onenote.svg

```diff
@@ -0,0 +1,24 @@
+<svg version="1.1" id="Livello_1" xmlns:x="http://ns.adobe.com/Extensibility/1.0/" xmlns:i="http://ns.adobe.com/AdobeIllustrator/10.0/" xmlns:graph="http://ns.adobe.com/Graphs/1.0/" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 2744.833 2553.333" enable-background="new 0 0 2744.833 2553.333" xml:space="preserve">
```

Comment:
Can you move this file elsewhere. I know it's an SVG but I don't like adding images to the repo.